content/english/articles/effective-gnops/snapshot-nodes/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Snapshot nodes: Backups and fast syncronization"
+publishDate: 2025-01-29T14:00:00-01:00
+translationKey: "snapshot-nodes"
+tags: [ "snapshot", "architecture", "nodes" ]
+level: Advanced
+author: albttx
+summary: Snapshot nodes enable rapid blockchain synchronization by providing a block state, allowing new or recovering nodes to quickly start with the latest version of the network. This approach significantly reduces sync time and resource usage, making it easier for participants to join or maintain nodes in the Gno.land ecosystem.
+---
+
+Snapshot nodes are a crucial component in blockchain ecosystems, providing reliable infrastructure for rapid network synchronization. These snapshots allow validators and other network participants to quickly synchronize with the network by downloading the blockchain state. This article explores how to create snapshots and where to store them.
+
+## What is a Snapshot Node?
+
+A snapshot node is essentially a standard Gno.land node with an additional feature: a script that periodically stops the node, backs up the data folder, and restarts the node. This process typically occurs every few hours, depending on the configuration.
+
+## Types of Snapshots
+
+There are two main types of nodes that can produce snapshots:
+
+1. **Full Node**: Contains the entire state of the blockchain since block 1. While these snapshots can be large, they allow users to recover data from any block. It's commonly called an **archive node**.
+
+2. **Pruned Node**: Contains partial states of the blockchain. Currently, pruned nodes are not possible in Gno.land, but there is an ongoing issue ([gnolang/gno #3637](https://github.com/gnolang/gno/issues/3637)) addressing this feature.
+
+## Creating and Managing Snapshots
+
+Here's a step-by-step guide on how to run a snapshot node:
+
+### Requirements
+
+- A [Gno.land](https://github.com/gnolang/gno) node
+- [lz4](https://github.com/lz4/lz4) for high compression
+- [Docker](https://docker.com)
+
+For this example, we'll run our Gno.land node inside a Docker container using Docker Compose.
+
+### Step 1: Set up the Docker Compose file
+
+Create a `docker-compose.yml` file with the following content:
+
+```yaml
+services:
+  gnoland:
+    image: "ghcr.io/gnolang/gno/gnoland"
+    command: ["start", "--lazy"]
+    ports:
+      - 26657:26657 # ensure RPC listens on 0.0.0.0
+    volumes:
+      - ./gnoland-data:/gnoland-data
+```
+
+### Step 2: Start the node
+
+```bash
+docker compose up -d
+```
+
+### Step 3: Stop the node and backup the latest height
+
+```bash
+export LATEST_HEIGHT=$(curl -s localhost:26657/status | jq -r '.result.sync_info.latest_block_height') \
+  && docker compose stop
+```
+
+### Step 4: Compress the data folder
```

Comment:
it would be nice if you can explain which folder we are exactly taking snapshot of, what these folders represent and why these are sufficient to create a valid snapshot

Reply:
Correct, i've added a small explanation